kernel/include/stdint.h

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
I've added this file to handle the basic definition of `integer` types, I've base my file in the standard `stdint.h`, if we have an independent OS, it has to have all headers and declarations using a non-standard libc.